app/Http/Controllers/AppController.php

```diff
@@ -21,18 +21,19 @@ public function get_streams(Request $request){
         $success_response['status_code'] = '1';
 
 
-        if(!$request->has('user_id')){
-            $error_response['status']='Error : user_id not received.';
+        if(!$request->has('rollNo')){
```

Comment:
This requires a change in app side also as this is consumed by app.

Reply:
Yep. App backend changed already